src/ophyd_async/testing/_one_of_everything.py

```diff
@@ -53,10 +57,11 @@ def float_array_signal(
         ],
         dtype=dtype,
     )
-    return soft_signal_rw(Array1D[dtype], value)
+    return soft_signal_rw(Array1D[dtype], value, name)
 
 
-class OneOfEverythingDevice(Device):
+class OneOfEverythingDevice(StandardReadable):
+    # make a detector to test assert_configuration
     def __init__(self, name=""):
         self.int = soft_signal_rw(int, 1)
         self.float = soft_signal_rw(float, 1.234)
```

Comment:
Could you do this in the native StandardReadable way...
```suggestion
        with self.add_children_as_readables(Format.CONFIG_SIGNAL):
            self.int = soft_signal_rw(int, 1)
            self.float = soft_signal_rw(float, 1.234)
            ...
```